src/components/deals/details/DealDetails.tsx

```diff
@@ -11,7 +11,7 @@ interface DealDetailsProps {
 }
 export default function DealDetails({ deal }: DealDetailsProps) {
   return (
-    <div className="text-white">
+    <div className="sticky top-0 pt-2 text-white">
```

Comment:
Unfortunately, we can't add this directly to the `DealDeatils` component because it is used on the individual deal page route as well. Could you look to move `sticky` to the parent container inside of the `/deals/add` route?

Reply:
You'll also need to add `items-start` class to the containing div as well as a little buffer on the top using `top-10`
![CleanShot 2024-09-23 at 11 53 38](https://github.com/user-attachments/assets/f08736a0-488e-47da-baec-1e45ce598281)